Sources/CocoaLumberjack/include/CocoaLumberjack/DDLog.h

```diff
@@ -845,7 +845,7 @@ typedef NS_OPTIONS(NSInteger, DDLogMessageOptions){
 @property (readonly, nonatomic) NSString *fileName;
 @property (readonly, nonatomic, nullable) NSString * function;
 @property (readonly, nonatomic) NSUInteger line;
-@property (readonly, nonatomic, nullable) id tag;
+@property (readonly, nonatomic, nullable) id representedObject;
```

Comment:
Simply renaming it like that is a breaking change (which is probably worse than the current situation, because it breaks almost *all* of the projects using CocoaLumberjack (versus just the ones having issues with code that calls `tag` on an arbitrary object without further classifying it).
At the very least we need to have a deprecated declaration of the old property around. I'm not sure, though, whether this will solve the selector conflict. If the ObjC compiler does treat deprecated declarations similar to the Swift compiler, it might. Otherwise it'll take some time until we can really remove the `tag` property altogether.

Reply:
I like the idea with the define!